<ui version="4.0" >
 <class>QgsDelimitedTextPluginGuiBase</class>
 <widget class="QDialog" name="QgsDelimitedTextPluginGuiBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Create a Layer from a Delimited Text File</string>
  </property>
  <property name="windowIcon" >
   <iconset>
    <normaloff/>
   </iconset>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>9</number>
   </property>
   <property name="topMargin" >
    <number>9</number>
   </property>
   <property name="rightMargin" >
    <number>9</number>
   </property>
   <property name="bottomMargin" >
    <number>9</number>
   </property>
   <property name="horizontalSpacing" >
    <number>6</number>
   </property>
   <property name="verticalSpacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" colspan="3" >
    <widget class="QLabel" name="txtHeading" >
     <property name="font" >
      <font>
       <family>Sans Serif</family>
       <pointsize>20</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text" >
      <string>Delimited Text Layer</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <widget class="QTextEdit" name="teInstructions" >
     <property name="readOnly" >
      <bool>true</bool>
     </property>
     <property name="html" >
      <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Verdana'; font-size:10pt; font-weight:400; font-style:normal;">
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;/p>&lt;/body>&lt;/html></string>
     </property>
    </widget>
   </item>
   <item row="5" column="2" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="textLabel2" >
       <property name="text" >
        <string>&lt;p align="right">X field&lt;/p></string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbXField" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip" >
        <string>Name of the field containing x values</string>
       </property>
       <property name="whatsThis" >
        <string>Name of the field containing x values. Choose a field from the list. The list is generated by parsing the header row of the delimited text file.</string>
       </property>
       <property name="editable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="textLabel2_2" >
       <property name="text" >
        <string>&lt;p align="right">Y field&lt;/p></string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbYField" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip" >
        <string>Name of the field containing y values</string>
       </property>
       <property name="whatsThis" >
        <string>Name of the field containing y values. Choose a field from the list. The list is generated by parsing the header row of the delimited text file.</string>
       </property>
       <property name="editable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="2" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="textLabel1" >
       <property name="text" >
        <string>Delimited text file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QgsFileDropEdit" name="txtFilePath" >
       <property name="toolTip" >
        <string>Full path to the delimited text file</string>
       </property>
       <property name="whatsThis" >
        <string>Full path to the delimited text file. In order to properly parse the fields in the file, the delimiter must be defined prior to entering the file name. Use the Browse button to the right of this field to choose the input file.</string>
       </property>
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnBrowseForFile" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="toolTip" >
        <string>Browse to find the delimited text file to be processed</string>
       </property>
       <property name="whatsThis" >
        <string>Use this button to browse to the location of the delimited text file. This button will not be enabled until a delimiter has been entered in the &lt;i>Delimiter&lt;/i> box. Once a file is chosen, the X and Y field drop-down boxes will be populated with the fields from the delimited text file.</string>
       </property>
       <property name="text" >
        <string>Browse...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="2" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="textLabel1_2" >
       <property name="text" >
        <string>Layer name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtLayerName" >
       <property name="toolTip" >
        <string>Name to display in the map legend</string>
       </property>
       <property name="whatsThis" >
        <string>Name displayed in the map legend</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="7" column="2" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item rowspan="7" row="1" column="0" >
    <widget class="QLabel" name="pixmapLabel1" >
     <property name="pixmap" >
      <pixmap resource="delimited_text.qrc" >:/delimited_text.png</pixmap>
     </property>
     <property name="scaledContents" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="2" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Sample text</string>
     </property>
     <layout class="QGridLayout" >
      <property name="leftMargin" >
       <number>9</number>
      </property>
      <property name="topMargin" >
       <number>9</number>
      </property>
      <property name="rightMargin" >
       <number>9</number>
      </property>
      <property name="bottomMargin" >
       <number>9</number>
      </property>
      <property name="horizontalSpacing" >
       <number>6</number>
      </property>
      <property name="verticalSpacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <widget class="QTextEdit" name="txtSample" >
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item rowspan="7" row="1" column="1" >
    <widget class="Line" name="line1" >
     <property name="frameShape" >
      <enum>QFrame::VLine</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="2" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="textLabel3" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Delimiter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtDelimiter" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>32767</width>
         <height>32767</height>
        </size>
       </property>
       <property name="toolTip" >
        <string>Delimiter to use when splitting fields in the text file. The delimiter can be more than one character.</string>
       </property>
       <property name="whatsThis" >
        <string>Delimiter to use when splitting fields in the delimited text file. The delimiter can be 1 or more characters in length.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="delimiterPlain" >
       <property name="toolTip" >
        <string>The delimiter is taken as is</string>
       </property>
       <property name="whatsThis" >
        <string>The delimiter is taken as is</string>
       </property>
       <property name="text" >
        <string>Plain characters</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="delimiterRegexp" >
       <property name="toolTip" >
        <string>The delimiter is a regular expression</string>
       </property>
       <property name="whatsThis" >
        <string>The delimiter is a regular expression</string>
       </property>
       <property name="text" >
        <string>Regular expression</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>61</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>QgsFileDropEdit</class>
   <extends>QLineEdit</extends>
   <header>qgsfiledropedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>txtLayerName</tabstop>
  <tabstop>txtDelimiter</tabstop>
  <tabstop>txtFilePath</tabstop>
  <tabstop>btnBrowseForFile</tabstop>
  <tabstop>cmbXField</tabstop>
  <tabstop>cmbYField</tabstop>
 </tabstops>
 <resources>
  <include location="delimited_text.qrc" />
 </resources>
 <connections/>
</ui>
